src/types/mod.rs

```diff
@@ -2238,6 +2238,32 @@ pub struct NodeInfo {
     pub details:      NodeDetails,
 }
 
+/// Information of a baker for a certain reward period.
+#[derive(Debug)]
+pub struct BakerRewardPeriodInfo {
+    /// Metadata for the baker.
```

Comment:
You are introducing a new term here "metadata" that does not really explain anything. Rephrase please.

Reply:
In the proto file there is a better line
```
/// The baker id and public keys for the baker.
```